includes/core/classes/class-venue.php

```diff
@@ -138,6 +139,22 @@ public function register_post_type(): void {
 		);
 	}
 
+	public function sanitize_float( $value ) {
```

Comment:
Probably good to move some of these sanitization utilities to Utility class where they can be used in various places that needs same kind of sanitization.